docs/source/examples/rshadows_tutorial.md

```diff
@@ -147,43 +144,45 @@ def cirq_executor(
 ```
 
 ## 2. Pauli Twirling Calibration
+
 ### 2.1 PTM Representation
-The PTM (Pauli Transfer Matrix) or Liouville representation provides a vector representation for all linear operators $\mathcal{L}(\mathcal{H}_d)$ on an $n$-qubit Hilbert space $\mathcal{H}_d$ (where $d = 2^n$). This representation uses the normalized Pauli operator basis $\sigma_a=P_a/\sqrt{d}$, with $P_a$ being the standard Pauli matrices.
 
+The PTM (Pauli Transfer Matrix) or Liouville representation provides a vector representation for all linear operators $\mathcal{L}(\mathcal{H}_d)$ on an $n$-qubit Hilbert space $\mathcal{H}_d$ (where $d = 2^n$). This representation uses the normalized Pauli operator basis $\sigma_a=P_a/\sqrt{d}$, with $P_a$ being the standard Pauli matrices.
 
 ```{code-cell} ipython3
 from mitiq.utils import operator_ptm_vector_rep
 
 operator_ptm_vector_rep(cirq.I._unitary_() / np.sqrt(2))
 ```
 
-### 2.2 Pauli Twirling of Quantum Channel and Pauli Fidelity:
+### 2.2 Pauli Twirling of Quantum Channel and Pauli Fidelity
+
 The classical shadow estimation involves Pauli twirling of a quantum channel represented by $\mathcal{G} \subset U(d)$, with PTM representation $\mathcal{U}$. This twirling allows direct computation of $\widehat{\mathcal{M}}$ for the noisy channel $\Lambda$:
 \begin{equation}
-\widehat{\mathcal{M}} = \mathbb{E}_{\mathcal{G}}[\mathcal{U}^\dagger\mathcal{M}_z\Lambda\mathcal{U}]
+\widehat{\mathcal{M}} = \mathbb{E}*{\mathcal{G}}[\mathcal{U}^\dagger\mathcal{M}_z\Lambda\mathcal{U}]
 \end{equation}
 Local Clifford group projections are given by:
 \begin{equation}
-\Pi_{b_i}=\left\{
+\Pi*{b_i}=\left\{
```

Comment:
If we decide to keep this tool, we want to make sure it does not make changes to LaTex equations.